sslSniffer/sslSnifferTest/snifftest.c

```diff
@@ -38,6 +38,10 @@
     #define WOLFSSL_SNIFFER
 #endif
 
+#ifdef THREADED_SNIFFTEST
```

Comment:
Can we just use `HAVE_PTHREAD` instead. Something like this:
```#if defined(HAVE_PTHREAD) && !defined(NO_THREADED_SNIFFTEST)
#define THREADED_SNIFFTEST
#endif
```

Reply:
And make it the default case whenever pthread is available?

Reply:
Yes if you add this above then it will do that:

```
#if defined(HAVE_PTHREAD) && !defined(NO_THREADED_SNIFFTEST)
#define THREADED_SNIFFTEST
#endif
```

Reply:
Done